cypress.config.js

```diff
@@ -2,6 +2,7 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://react-redux.realworld.io',
```

Comment:
The baseURL should be set in the Cypress configuration file under 'baseUrl' property, not under 'e2e'

Reply:
![image](https://github.com/mate-academy/qa_cypress_e2e_intro_adv/assets/84145664/c427c4e4-2583-4ba8-9983-35418117c874)
if you set baseUrl in the Cypress configuration file, it will not work.